Q3DC/Q3DC.py

```diff
@@ -1379,14 +1379,21 @@ def computeAngles(self, line1, line2, states):
         return result
 
     def computeLinePoint(self, lineA, lineB, point):
-        closestPoint = np.zeros(3)
-        t = vtk.reference(0)
-        squared = vtk.vtkLine.DistanceToLine(point, lineA, lineB, t, closestPoint)
-        norm = np.sqrt(squared)
+        # Adapted from vtkLine::DistanceToLine(const double x[3], const double p1[3], const double p2[3])
+        pointLineA = point - lineA
+        lineALineB = lineA - lineB
 
-        delta = point - closestPoint
-        norm = np.linalg.norm(delta)
+        den = np.linalg.norm(lineALineB)
+        if den != 0.0:
+          lineALineB = lineALineB / den
+        else:
+          return [0., 0., 0., np.sqrt(np.dot(pointLineA, pointLineA))]
+
+        proj = np.dot(pointLineA, lineALineB)
+
+        norm = np.sqrt(np.dot(pointLineA, pointLineA) - proj * proj)
 
+        delta = point - proj
```

Comment:
```suggestion
        delta = point - proj

        norm = np.linalg.norm(delta)
```

I think this would be easier to understand. Everything looks good to me though.

Reply:
It would also be ideal to add a simple test for this function (e.g self test)

Reply:
I am writing a test for the function, and my suggested implementation is not correct (neither is the originally suggested one.)

I am working on a correction.

Reply:
> I am working on a correction.

Thanks :pray: